test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
- [ ] You have done an excellent job with your project but there is an issue with this file because it was ignored and it should not be in the PR. I think the problem is because you tracked it (when making a commit) before ignoring it. You could try [this solution](https://stackoverflow.com/questions/1274057/how-can-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitign) :wink:

Reply:
Thank you for your review, I have resolved this based on your suggested solution.